.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,40 @@
+name: End to end tests
+
+on:
+  deployment_status:
+
+concurrency:
+  group: ${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  e2e_tests:
+    name: E2E tests
+    if: github.event.deployment_status.state == 'success' && endsWith(github.event.deployment_status.environment, 'blockprotocol')
```

Comment:
this line has changed since the previous version to add a check for the environment. See example status events in PR description – it will be something like "Preview – blockprotocol" or "Preview – blockprotocol-design-system"